pom.xml

```diff
@@ -233,13 +244,11 @@
 
   <properties>
     <localTeamServerUrl>http://localhost:19080/Contrast/api</localTeamServerUrl>
-    <stagingTeamServerUrl>http://teamserver-334.internal.contsec.com/Contrast/api
```

Comment:
eww... ive never even heard of this..

Reply:
This is old. We used to deploy each release to its own subdomain, and it was only accessible from our old VPN. Clearly, this hasn't been used in a while.